data/repositories.toml

```diff
@@ -803,5 +803,6 @@ repositories = [
   'github.com/zulip/zulip',
   'github.com/ZupIT/charlescd',
   'github.com/ZupIT/ritchie-cli',
-  'github.com/ZupIT/ritchie-formulas'
+  'github.com/ZupIT/ritchie-formulas',
+  'github.com/Mom0aut/DynamicRelations'
```

Comment:
Hey @Mom0aut - Thanks for creating the PR. 
To ensure quality and standard, we require at least 10 contributors to add your repository to Good First Issue.
You can check the criteria [here](https://github.com/deepsourcelabs/good-first-issue#adding-a-new-project).